### ISSUE TITLE
How postgresql Db can be exported from docker? Or how we can get the path for products.csv.gz to download. Any suggestions

### ISSUE THREAD

**Issue by amitlals** [infracost/infracost#2348]

**Comment by amitlals** [comment]
> hi @amitlals, I'm unsure what exactly your asking for here. Is it that you want to connect to the cloud pricing API database locally?

Yes! Thanks for your response. I am exploring if I could get the zip file directly before it goes Postgresdb in docker. Not sure if works? Cheers!

**Comment by hugorut** [comment]
Hi @amitlals you can just run the `dataDownload` command https://github.com/infracost/cloud-pricing-api/blob/60f7632eff19f0cba93653c2495cddc5bad2dc06/package.json#L14 which will download the file locally.

**Comment by amitlals** [comment]
> Hi @amitlals you can just run the `dataDownload` command https://github.com/infracost/cloud-pricing-api/blob/60f7632eff19f0cba93653c2495cddc5bad2dc06/package.json#L14 which will download the file locally.

Thanks again! Pls elaborate how this command works, is it works infacost EXE? or something? sorry for unaware here. Thanks

**Comment by alikhajeh1** [comment]
@amitlals you can follow [this guide](https://github.com/infracost/cloud-pricing-api/blob/master/CONTRIBUTING.md) and after step 5 you can run the command that Hugo mentioned (`npm run data:download`), which downloads the file you want. Notice how this is part of the Cloud Pricing API, not part of the Infracost CLI repo.
